jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -224,7 +224,9 @@ private static final class AnnotationVars {
 /*[IF JAVA_SPEC_VERSION >= 11]*/
 	private static ReflectionFactory reflectionFactory;
 /*[ENDIF] JAVA_SPEC_VERSION >= 11 */
-	
+
+	private String toStringName;
```

Comment:
This can be grouped with `classNameString`. Following the same pattern, it should be declared `transient`.